src/omv/modules/py_tf.c

```diff
@@ -578,9 +479,10 @@ STATIC void py_tf_segment_output_callback(void *callback_data,
 }
 
 STATIC mp_obj_t py_tf_segment(uint n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
```

Comment:
It outputs [h][w][c]. It could be replaced with invoke() and then a custom callback that makes an image. However, this should be done in another PR.

I'll need to think about how this should work... we don't want to end up in a situation where the time it takes the build the output image is so slow in Python that you can't get over 1 FPS when we have new faster chips. I think the best thing to do here is wait until I have a test network. Then I can try to figure out a pattern that's flexible and has good performance.

Reply:
If it outputs a small image/map, we can pass it back to some C code along with the main image to do the processing. A function in py_image for example.

Reply:
Okay, will redo the segment code once we have a network to test with.